cookbooks/bcpc-hadoop/attributes/hdfs.rb

```diff
@@ -118,28 +118,62 @@
     '30000'
 end
 
-# Default HDFS Directory Quotas
-default['bcpc']['hadoop']['hdfs'].tap do |hdfs|
-  hdfs['user']['space_quota'] = '30G'
-  hdfs['groups']['space_quota'] = '50G'
+# Default HDFS User Directory Configuration
+default['bcpc']['hadoop']['hdfs']['user'].tap do |user|
+  user['home'] = '/user'
+  user['group'] = 'supergroup'
+  user['perms'] = '0700'
+  user['space_quota'] = '30G'
+  user['ns_quota'] = 'NO_QUOTA'
+  user['dirinfo'] = {
+    ## Example dir_info entry:
+    # user_dirname: {
+    #   owner: 'user',
+    #   group: 'group',
+    #   perms: '0700',
+    #   space_quota: '5T',
+    #   ns_quota: '500000'
+    # }
+  }
 end
 
-# Default HDFS Projects Configuration
+# Default HDFS Groups Directory Configuration
+default['bcpc']['hadoop']['hdfs']['groups'].tap do |groups|
+  groups['home'] = '/groups'
+  groups['owner'] = 'hdfs'
+  groups['perms'] = '0770'
+  groups['space_quota'] = '50G'
+  groups['ns_quota'] = 'NO_QUOTA'
+  groups['dirinfo'] = {
+    ## Example dir_info entry:
+    # group_dirname: {
+    #   owner: 'user',
+    #   group: 'group',
```

Comment:
Clarify `group` will always be set to the group we pass, and remove this example from the comment.